drivers/adc/ad796x/README.rst

```diff
@@ -0,0 +1,218 @@
+AD796X no-OS driver
+====================
+
+Supported Devices
+-----------------
+
+`AD7960 <https://www.analog.com/en/products/ad7960.html>`_
+
+Overview
+--------
+
+The EVAL-AD7960FMCZ is a fully featured evaluation Kit designed to demonstrate
+the performance of the low power 18-bit,5 MSPS PulSAR® Differential ADC AD7960.
+
+This board operates in conjunction with the Zedboard. The evaluation software
+is provided to enable the user to perform detailed analysis of the AD7960's
+performance. The technical user guide includes the detailed description of
+the operation and set up of the evaluation board and software when operated
+with the System Demonstration Platform board.
+
+
+AD796X Device Configuration
+----------------------------
+
```

Comment:
looks like device configuration documentation is missing...
in general, I think it will be useful to also describe the available APIs, just in case someone checks the documentation before the code... there are various examples here: http://analogdevicesinc.github.io/no-OS/drivers_doc.html

Reply:
i took the example from https://analogdevicesinc.github.io/no-OS/drivers/adm1177.html
where "driver initialization " is a subtopic of "device configuration "

in fact there is no device configuration for this driver. only driver initialization and reading data,
which i mention, but perhaps not clearly enough?

Reply:
It is not clear that driver initialization is a subtopic of device configuration. they are using the same indentation. use `^^^^`, instead of `----` for driver initialization if that is the case

Reply:
ok , i  just followed what was done in the other docs: 

```
AD405X Device Configuration
---------------------------

Driver Initialization
---------------------

In order to be able to use the device, you will have to provide the support for
the communication protocol (SPI) as well as 3 external GPIOs for the CNV pin and two
```

```
ADM1177 Device Configuration
----------------------------

Driver Initialization
---------------------

In order to be able to use the device, you will have to provide the support for
the communication protocol (I2C) as mentioned above.

The first API to be called is **adm1177_init**. Make sure that it returns 0,
which means that the driver was initialized correctly.

```